service/init.go

```diff
@@ -333,7 +334,7 @@ func (amf *AMF) Start() {
 	}
 
 	signalChannel := make(chan os.Signal, 1)
-	signal.Notify(signalChannel, os.Interrupt, syscall.SIGTERM)
+	signal.Notify(signalChannel, syscall.SIGHUP, syscall.SIGQUIT, syscall.SIGTERM, syscall.SIGINT, syscall.SIGSEGV, syscall.SIGCHLD)
```

Comment:
Which signal is received? Is it good idea to catch SIGSEGV as well ?. Do we send de-registration even during seg mentation fault ?